CentrifugeiOS/CentrifugeiOS.h

```diff
@@ -15,4 +15,4 @@ FOUNDATION_EXPORT const unsigned char CentrifugeiOSVersionString[];
 
 // In this header, you should import all the public headers of your framework using statements like #import <CentrifugeiOS/PublicHeader.h>
 
-
+#import <CentrifugeiOS/CommonCryptoBridge.h>
```

Comment:
Why do you want this header be public?

Reply:
To make it visible for swift code. No other reason to do that.

Reply:
I removed this line, example project works 🤔

Reply:
Let me check then